pkg/kapp/cmd/app/inspect.go

```diff
@@ -23,9 +23,10 @@ type InspectOptions struct {
 	ResourceFilterFlags cmdtools.ResourceFilterFlags
 	ResourceTypesFlags  ResourceTypesFlags
 
-	Raw    bool
-	Status bool
-	Tree   bool
+	Raw               bool
+	Status            bool
+	Tree              bool
+	ShowManagedFields bool
```

Comment:
carvel tools typically do not use verb in the flag name. so in this case ManagedFields=true/false would be more consistent

Reply:
Should the flag also be renamed to managed-fields (from show-managed-fields, tried to use the same flag that  kubectl uses.)?